conbench/templates/run.html

```diff
@@ -189,7 +189,7 @@ <h6>Top Outliers</h6>
 } );
 
 enable_search_query_string('#benchmarks');
-specific_column_search('#benchmarks', [1]);
+dropdown_column_search('#benchmarks', [1]);
```

Comment:
Should this be `column_search_implementation`? That would end in there being a dropdown if the number of elements is small(ish) right?

Reply:
The functions `dropdown_column_search` & `free_text_column_search` works for now in standalone mode. Didn't make the change here yet (while testing)

Shall update :)